docs/api/latest.md

```diff
@@ -0,0 +1,122 @@
+# API Docs - v1.0.0-SNAPSHOT
```

Comment:
Same

Reply:
Fixed with commit: `Fix some suggested issues in PR#1.` 8a89f22